Unbox.swift

```diff
@@ -696,12 +691,64 @@ private extension Unboxer {
         
         return unboxed
     }
+
+    /// Unbox a JSON dictionary into a model `T`, while optionally using a contextual object. Throws `UnboxError`.
+    static func performUnboxingWithDictionary<T: Unboxable>(dictionary: UnboxableDictionary, context: Any? = nil) throws -> T {
```

Comment:
Could you move this so that it's together with the other `static` functions? Also, the documentation can be removed (since it's private), and remove the default value for the `context` parameter.